slack_cleaner/cli.py

```diff
@@ -104,11 +104,8 @@ def clean_channel(channel_id, time_range, user_id=None, bot=False):
             if m['type'] == 'message':
                 # Delete user messages
                 if m.get('user'):
-                    # No matter whos message, just delete
-                    if user_id is None:
```

Comment:
My bad.

Didn't catch your idea in the beginning. Now I must admit it's a critical bug and also a silly design flaw of argument combination. :scream: I'm going to merge this and make some breaking changes in the next release.

Also, thanks for your suggestion on Slack's *message retention*, though it's a paid plan feature.